packages/conjure-client/src/fetchBridge/fetchBridge.ts

```diff
@@ -188,13 +188,31 @@ export class FetchBridge implements IHttpApiBridge {
         return query.length > 0 ? `?${query.join("&")}` : "";
     }
 
+    private buildFormString(parameters: IHttpEndpointOptions) {
```

Comment:
isn't this basically the same as `buildQueryString`? `[].join("&") === ""`

Reply:
would prefer if we refactored buildQueryString to take in the arguments to construct a query string with instead of duplicating the logic here